components/molecule/tabs/src/index.js

```diff
@@ -46,7 +46,9 @@ const MoleculeTabs = ({variant, type, children, onChange}) => {
   return (
     <div className={className}>
       <ul className={CLASS_SCROLLER}>{extendedChildren}</ul>
-      <div className={CLASS_CONTENT}>{activeTabContent}</div>
+      {activeTabContent ? (
+        <div className={CLASS_CONTENT}>{activeTabContent}</div>
+      ) : null}
```

Comment:
```suggestion
      {activeTabContent && (
        <div className={CLASS_CONTENT}>{activeTabContent}</div>
      )}
```

no need to return a value, could you remove the ternary?

Reply:
This ternary exactly do that. Use ternaries could cause some undesirable problems because if first operation is not "true" React can render the variable value (for example: 0 or false)

I recommend you an interesting article about that https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
Agree with @rmoralp

Reply:
Thanks @rmoralp!! Excellent article!!  😍 😍

I usually use logical operators a lot and to avoid rendering "0 " them I do the operation with as `!!foo.length`

The ternaries as a null return I use them  in the first level of rendering of the component.

Thanks!